apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -740,7 +740,10 @@ export function buildMachine(
                     'resetContext',
                     'endCardlessVoterAuth',
                   ],
-                  target: 'accepting_paper',
+                  // Return to the initial poll worker screen so that the poll worker has to
+                  // explicitly choose whether they want to start a new session with a blank sheet
+                  // or insert a preprinted ballot again
+                  target: 'not_accepting_paper',
```

Comment:
See `Commit 3: Addressing crash involving preprinted ballot insertion edge case` in the PR description for more details